packages/@aws-cdk/aws-stepfunctions-tasks/test/stepfunctions/integ.invoke-activity.ts

```diff
@@ -42,6 +42,13 @@ class InvokeActivityStack extends cdk.Stack {
     const finalStatus = new tasks.StepFunctionsInvokeActivity(this, 'Get Final Job Status', {
       activity: checkJobActivity,
       inputPath: '$.guid',
+      parameters: {
+        'input.$': '$',
+        'stringArgument': 'inital-task',
+        'numberArgument': 123,
+        'booleanArgument': true,
+        'arrayArgument': ['a', 'b', 'c'],
+      },
```

Comment:
Could you elaborate on that further? Because in this test are some invoked activities without the parameters and I dont quite understand what do you mean exactly.

Should I do a different integration test?

Reply:
My bad, you're absolutely right. I've missed the other activities. 🤦‍♂️

Let's add a line with `JsonPath` to the test case and we are good!